course_content/notebooks/numpy_intro.ipynb

```diff
@@ -263,7 +275,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Exercise\n",
+    "### Exercise 1\n",
```

Comment:
@dkillick is making a PR (#137) which changes "Explore: creating arrays" the first exercise. Once his PR is merged it would be worth rebasing and re-indexing the exercises from that one.